src/xr_3da/entry_point.cpp

```diff
@@ -76,16 +77,17 @@ int APIENTRY WinMain(HINSTANCE inst, HINSTANCE prevInst, char* commandLine, int
 int main(int argc, char *argv[])
 {
     int result = EXIT_FAILURE;
-
+    static std::locale locale("");
```

Comment:
Wait.. Is this locale even used?

Reply:
I also don't understand the aim of this line. Remove it or at least write a comment.

Reply:
It sets default locale for current system, some how i remember this was a bug in runtime, when default locale did not sets when program starts. But this is not my addition, I believe that the one who added this line faced this problem.

Reply:
Looks like a most of changes of this file are NOT the fixes for bugs but the improvements of the style. Maybe split a real bugfixes and style improvements to different commits (or even PRs)?